README.rst

```diff
@@ -39,63 +39,80 @@
         :target: https://github.com/SwissDataScienceCenter/renga-python/blob/master/LICENSE
 
 A Python library for the `Renga collaborative data science platform
-<https://github.com/SwissDataScienceCenter/renga>`_. It lets you perform any action with
-``renga`` command or from withing Python apps - create projects, manage
-buckets, track files, run containers, etc.
+<https://github.com/SwissDataScienceCenter/renga>`_. It lets you perform any
+action with ``renga`` command or from withing Python apps - create projects,
+manage buckets, track files, run containers, etc.
 
 
 **NOTE**:
-   ``renga-python`` is the python library for Renga that provides an SDK and a 
-   command-line interface (CLI). It *does not* start the Renga platform itself - 
-   for that, refer to the Renga docs on `running the platform 
+   ``renga-python`` is the python library for Renga that provides an SDK and a
+   command-line interface (CLI). It *does not* start the Renga platform itself -
+   for that, refer to the Renga docs on `running the platform
    <https://renga.readthedocs.io/en/latest/user/setup.html>`_.
 
-**This is an experimental developer preview release.**
+**This is the development branch of `renga-python` and should be considered
+highly volatile.**
 
 Installation
 ------------
 
-The latest release is available on PyPI and can be installed using
-``pip``:
+Clone the repository and install using ``pip``:
```

Comment:
I would not change the installation process in development branch. We can push dev releases to PyPI too  and then one need to use `pip install --pre renga`.

Reply:
That's fine, we'll add pip instructions when that's in place -- for right now, this is the way to install it.